src/unstructured_extractor.py

```
@@ -0,0 +1,1203 @@
+    # Headers/footers are outside hierarchy
+    if category in ["Header", "Footer", "PageBreak"]:
+        return (category, 0.0)
+
+    # Tables
+    if category == "Table":
+        return ("Table", 0.6)
+
+    return ("Unknown", 0.3)
+
+
+def detect_hierarchy_generic(elements: List[Element], config: ExtractionConfig) -> List[Dict]:
```

Ruff (0.14.3)

Unused function argument: `config`

(ARG001)